src/components/CardContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react"
+
+import elementStyles from "styles/isomer-cms/Elements.module.scss"
+
+export const CardContainer = ({ cardTitle, children, isError, onClose }) => {
+  const [shouldDisplay, setShouldDisplay] = useState(false)
+
+  return (
+    <div
+      className={`${elementStyles.card} ${
+        !shouldDisplay && isError ? elementStyles.error : ""
+      }`}
+    >
+      <div className={elementStyles.cardHeader}>
+        <h2>{cardTitle}</h2>
+        <button
+          className="pl-3"
+          type="button"
+          onClick={() => {
+            setShouldDisplay(!shouldDisplay)
+            if (shouldDisplay && onClose) onClose()
```

Comment:
Thanks for catching - this is a functionality I introduced previously but meant to deprecate, done here ce05fcd9!

Though, in response to the above, the operation checks for whether `shouldDisplay === true` and `onClose` is defined, so it would run when it is already open and the user clicks to close it, which triggers `onClose()` to run (which contained any other cleanup effects I was trying to trigger. In retrospect, this was probably only working because `useState` in React is not a synchronous operation >.< lol . I'm glad we deprecated it.